core/domain/recommendations_jobs_one_off.py

```diff
@@ -124,3 +126,47 @@ def map(model):
     @staticmethod
     def reduce(key, values):
         yield (key, len(values))
+
+
+class CleanUpExplorationRecommendationsOneOffJob(
```

Comment:
Adding a note to confirm that this is done automatically in code base now: https://github.com/oppia/oppia/blob/develop/core/domain/exp_services.py#L769.

So, we don't need an additional fix. Though the ideal way to tackle this would be to have a fix for #10809